docs/advanced.rst

```diff
@@ -46,7 +63,6 @@ Another example is related to the `feature request #143
 at the beginning and the end of a file.  This use case is easy to implement
 through a custom wrapper for the PEP 8 library::
 
-  #!python
```

Comment:
> updates the heading format for consistency

I made "Configuring tests" a subsection of "Automated tests" intentionally, but I guess the whole "Advanced usage" guide is really about how to use pycodestyle in automated tests.